isos/bootstrap/bootstrap

```diff
@@ -63,19 +63,17 @@ if mount -t ext4 /dev/disk/by-label/containerfs ${MOUNTPOINT}; then
     # libdl.so.2 => /baz/lib/libdl.so.2 (0x00007f97fb84f000)
     # /lib64/ld-linux-x86-64.so.2 (0x00007f97fc929000)
 
-    mkdir ${MOUNTPOINT}/.tether/lib
-    mkdir ${MOUNTPOINT}/.tether/lib64
-    cp -a /sbin/*tables* ${MOUNTPOINT}/.tether/
+    ln -s lib64 ${MOUNTPOINT}/.tether/lib
+    mkdir -p ${MOUNTPOINT}/.tether/{lib64,usr/lib/iptables}
 
+    cp -a /sbin/*tables* ${MOUNTPOINT}/.tether/
     cp -a /lib/libm.* /lib/libm-* /lib/libgcc_s* /lib/libip*tc* /lib/libxtables* /lib/libdl* /lib/libc.so* /lib/libc-* ${MOUNTPOINT}/.tether/lib
     cp -a /lib64/ld-* ${MOUNTPOINT}/.tether/lib64
+    cp -r /usr/lib/iptables ${MOUNTPOINT}/.tether/usr/lib/
 
```

Comment:
Unfortunately, as @fdawg4l and I both found the offical stance for using netfilter for ipchains is "Don't. Use the iptables binary"